packages/@okta/vuepress-site/use_cases/mfa/index.md

```diff
@@ -144,10 +135,8 @@ this process is known as "Enrollment". Once a MFA token has been
 enrolled, we can then "Verify" that they actually have this token
 by asking them to answer a challenge using their token.
 
-The process attaching a factor to a user is similar for every type
-of factor that Okta supports. In the video we show how to attach a
-Google Authenticator factor as well as a phone as a factor using
-SMS. In this document we will only cover how to attach a Google
+The process attaching of a factor to a user is similar for every type
+of factor that Okta supports. In this document we will only cover how to attach a Google
```

Comment:
of factor that Okta supports. In this document we only cover how to attach a Google